src/frontend/main.go

```diff
@@ -157,6 +184,17 @@ func initStats(log logrus.FieldLogger, exporter *stackdriver.Exporter) {
 	}
 }
 
+func initMeter(log logrus.FieldLogger) *push.Controller {
```

Comment:
document the method
also perhaps worth renaming to initMetricsCollection or something along those lines so it's clear what it does.

Reply:
done